apps/cloudfront-functions/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    watch: false
+  },
+});
```

Comment:
I would bring back the configurations related to coverage that [were defined in the Jest configuration file](https://github.com/pagopa/b2b-portals/pull/113/files#diff-835c022019175b301a630592556b9b4c4034bbe201028093d47e5646b1858948R9-R14).

Reply:
the only issue about coverage as the jest configuration is that vitest doesn't support the same thing as jest. I tried to put it in, but the only few lines about coverage I was able to do was about provider and reporter (as you'll see from the new commit)

Reply:
The coverage should be also [configurable in vitest](https://vitest.dev/config/#coverage-thresholds-lines).

Reply:
pushed a new version